### PR TITLE
fix padding for container

### PR DESCRIPTION
**Pull request by thisIvanRepo**

### CODE REVIEW

File: src/css/base/container.css [hunk 1]
```diff
@@ -3,17 +3,20 @@
...
 }
 /* Styles for Tablet devices */
 @media only screen and (min-width: 768px) {
   .container {
     max-width: 768px;
+    padding: 0 32px;
   }
 }
 /* Styles for Desktop devices */
 @media only screen and (min-width: 1200px) {
```
Comment (ElephantFrontend) [inline src/css/base/container.css]: min-width: 1280px.
Change please. And check result.

File: src/css/base/container.css [hunk 2]
```diff
@@ -3,17 +3,20 @@
   min-width: 320px;
```
Comment (ElephantFrontend) [inline src/css/base/container.css]: width: 100%
max-width: 768px
padding: 0 20px;
margin: 0 auto;

File: src/css/base/container.css [hunk 3]
```diff
@@ -3,17 +3,20 @@
...
   max-width: 375px;
   margin: 0 auto;
-  padding: 0 16px;
+  padding: 0 20px;
 }
 /* Styles for Tablet devices */
 @media only screen and (min-width: 768px) {
   .container {
```
Comment (ElephantFrontend) [inline src/css/base/container.css:11]: remove max-width please;
add:
padding: 0 32px;

File: src/css/base/container.css [hunk 4]
```diff
@@ -3,17 +3,20 @@
...
     max-width: 768px;
+    padding: 0 32px;
   }
 }
 /* Styles for Desktop devices */
 @media only screen and (min-width: 1200px) {
   .container {
     max-width: 1200px;
```
Comment (ElephantFrontend) [inline src/css/base/container.css]: max-width: 1152px;
for some sections type of (about us, teachers...) we can add 
padding: 0 64px;

File: src/css/base/container.css (hunk 1 above)
Comment (thisIvanRepo) [reply src/css/base/container.css]: fixed

File: src/css/base/container.css (hunk 2 above)
Comment (thisIvanRepo) [reply src/css/base/container.css]: fixed

File: src/css/base/container.css (hunk 3 above)
Comment (thisIvanRepo) [reply src/css/base/container.css:11]: fixed

Comment (thisIvanRepo) [reply src/css/base/container.css:11]: fixed

**Pull request merged**